core/src/index.ts

```diff
@@ -27,5 +27,4 @@ export {
   popoverController,
   toastController,
 } from './utils/overlays';
-export { IonicSwiper } from './components/slides/IonicSwiper';
 export { IonicSlides } from './components/slides/IonicSlides';
```

Comment:
This should likely be removed as well with `components/slides` directory being deleted.

Reply:
I think `IonicSlides` is still there in the PR. This is the module that devs can use to get the same presets as the old `ion-slides`.